ducktape/cluster/remoteaccount.py

```diff
@@ -180,17 +180,21 @@ def create_file(self, path, contents):
     def _ssh_quiet(self, cmd, allow_fail=False):
         """Runs the command on the remote host using SSH and blocks until the remote command finishes.
 
-        If it succeeds, there is no output; if it fails the output is printed and the CalledProcessError is re-raised.
+        If it succeeds, there is no output; if it fails the output is printed
```

Comment:
I updated the docs for `ssh()`. For the rest, it's not relevant since they don't use `allow_fail=True` and we weren't making any guarantees about what they previously returned.